scenarios/kubernetes_kops_aws.py

```diff
@@ -135,15 +163,14 @@ def main(args):
       '-e', 'E2E_UP=%s' % args.up,
       '-e', 'E2E_TEST=%s' % args.test,
       '-e', 'E2E_DOWN=%s' % args.down,
-      '-e', 'KOPS_E2E_CLUSTER_NAME=%s' % args.cluster,
-      # AWS
-      '-e', 'KOPS_ZONES=%s' % zone,
-      '-e', ('E2E_OPT=--kops-cluster %s --kops-zones %s --kops-state %s --kops-nodes=4' %
-             (args.cluster, zone, args.state)),
-      '-e', 'KOPS_REGIONS=%s' % zone[:-1],
+      # Kops
+      '-e', ('E2E_OPT=--kops-cluster %s --kops-zones %s '
+             '--kops-state %s --kops-nodes=%s --kops-ssh-key %s' %
```

Comment:
It's not necessary, but I fixed it for consistency. (I moved it from `kops-e2e-runner` and it had no equals there, but it does here. So much for consistency between those. :) )